docs/store/log-artifacts.md

```diff
@@ -0,0 +1,85 @@
+(log-artifacts)=
+# Logging Artifacts
+To log artifacts that are not of kind model or dataset, use the {py:meth}`~mlrun.execution.MLClientCtx.log_artifact` method.
+You can apply this method to the project object or the context object when logging an artifact in runtime, for example a job.
+**In this section**
+- [Basic logging of an artifacts file](#basic-logging-of-an-artifacts-file)
+- [Log a Plotly object as an html file](#log-a-plotly-object-as-an-html-file)
+- [Logging Plotly artifacts](#logging-plotly-artifacts)
+- [Logging directory artifacts](#logging-directory-artifacts)
+
+**See also**
+- {ref}`working-with-data-and-model-artifacts`
+- {ref}`models`
+- {ref}`logging_datasets`
+- [Logging a Databricks response as an artifact](../runtimes/databricks.html#logging-a-databricks-response-as-an-artifact).
+## Basic logging of an artifacts file
+`log_artifact` can be used to log many kinds of files, for example `html`,`pkl` and more. This is the most general method of logging artifacts. 
+```{admonition} Tip
+Each object type requires a different way to convert it for logging the object to a file. This is just an example of one type.
+```
+To log an artifacts file, specify the local file path to the file using the `local_path`, or use the `body` to dump the object content.
+
+**Log with local path**
+```python
+    with open("file.txt","w") as f:
+        f.write("abc is 123")
+    
+    project.log_artifact(
+        "file-example",
+        local_path="file.txt",
+        labels={"Test": "label-test"},
+    )
+```
+
+**Log with body**
+```python
+    project.log_artifact(
+        "some-data",
+        body=b"abc is 123",
+        format="txt",
+        labels={"Test": "label-test"},
+    )
+```
```

Comment:
I think @theSaarco said once it is really not recommended to use `body` in `log_artifact`.

Reply:
That is correct, though for demo purposes it's ok to use a small body. The main issue was with `inline` artifacts, where the body was stored in the artifact metadata in the DB, causing size and scale issues. Nowadays we almost always upload the body as a file, unless the user specifically asks to make the artifact `inline`. 
Still, if we maintain this demo I would warn that it's just a demo and when using a fairly large body it's recommended to not use the `body` property but instead use a file.

Reply:
added comment